.github/CODEOWNERS

```diff
@@ -5,8 +5,8 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 # Note: This file is autogenerated. To make changes to the codeowner team, please update .repo-metadata.json.
 
-# @googleapis/yoshi-python @googleapis/aap-dpes are the default owners for changes in this repo
-*     @googleapis/yoshi-python @googleapis/aap-dpes
+# @googleapis/yoshi-python is the default owner for changes in this repo
+*     @googleapis/yoshi-python 
 
 # @googleapis/python-samples-reviewers @googleapis/aap-dpes are the default owners for samples changes
 /samples/   @googleapis/python-samples-reviewers @googleapis/aap-dpes
```

Comment:
Will the team still look after the samples?

Reply:
yes. there is a doc